packages/libp2p/test/connection-manager/index.spec.ts

```diff
@@ -365,7 +366,8 @@ describe('Connection Manager', () => {
       peerStore: stubInterface<PeerStore>(),
       transportManager: stubInterface<TransportManager>(),
       connectionGater: stubInterface<ConnectionGater>(),
-      events: new TypedEventEmitter()
+      events: new TypedEventEmitter(),
+      logger: defaultLogger()
```

Comment:
shouldn't it default to the `defaultLogger` as opposed to passing it each time?

Reply:
Could probably remove a bit of the duplication, but the component interface for the connection manager requires a logger so it doesn't need to worry about defaulting it.

Reply:
I've deduped the component setup.